### ISSUE TITLE
Can we switch to non-dev branch of pyqtgraph

### ISSUE THREAD

**Issue by markmac99** [CroatianMeteorNetwork/RMS#148]
The setup/requirements are using a development branch of pyqtgraph. Thats quite risky as it means that we could pick up untested changes, or RMS might be disabled if a bad checkin is made by one of the QT developers.
I will test if we can switch to the production branch
Also, we don't need a special build of python-dvr any more, this is now in pypi so I'll switch RMS over to that.  I've been using it on several of my machines for months. This will remove two special-cases from the environment

**Comment by dvida** [comment]
Thanks!

**Comment by markmac99** [comment]
pyqtgraph is only used by SkyFit2 and CustomPyqtgraphClasses.  Some minor changes are required to the latter, as QT dialogs have moved from QtGui to QtWidgets. python-dvr is only used by the camera control utility.
Both these modules are only supported on a Pi4.

I tested on a spare Pi4 with pyqtgraph-0.12.4. SkyFit2 runs correctly and no other issues were encountered. I have now upgraded UK0006 and will run overnight to verify no surprises.

I tested on Windows 10  with pyqtgraph-0.13.1 where SkyFit2 operates correctly and i had no issues with any other utility. 

I tested on a Pi3 where i confirmed that the previous version of pyqtgraph still installed from github. SkyFit2 doesn't run on a pi3 as noted above so i did not test that. Although CustomQtgraphClasses is referenced in FrameInterface, there's already a try/except block around it with a fallback to a tk class and i verified this works correctly.  

I've been using the 'official' version of Python-dvr on uk0006 for some months and have no concerns with it.
